### PR TITLE
Resources: Added new book

### PR DESCRIPTION
**Pull request by agcolom**

### CODE REVIEW

File: pages/resources.html [hunk 1]
```diff
@@ -115,6 +115,8 @@ <h3 style="font-size: 14px !important; color: #ddd; padding-bottom: 20px; border
 <tr class="booktable">
   <td class="book"><img src="/resources/books/jqmWebDevEssentials.png" width="105" style="margin-left:-5px" /></td>
   <td class="title">&#8220;<a href="http://www.packtpub.com/jquery-mobile-web-development-essentials-2e/book">jQuery Mobile Web Development Essentials - Second Edition</a>&#8221;<br /> by Raymond Camden and Andy Matthews,<br /> Packt Publishing</td>
+ <td class="book"><img src="/resources/books/MasteringjQueryMobile.jpg" width="105" style="margin-left:-5px" /></td>
```
Comment (arthurvr) [inline pages/resources.html]: this indentation seems wrong.

**Comment by arthurvr** [comment]
Is this how you wanted it to look? The layout seems broken with that last column.

resources___jquery_mobile_and__jquery__znc_jquery_com_

**Comment by agcolom** [comment]
It looked ok on my local setup. I'll double check this evening and fix the indentation. Thx

**Comment by agcolom** [comment]
@arthurvr hopefully this should fix it?

**Comment by agcolom** [comment]
@arschmitz @gabrielschulhof @jaspermdegroot Anyone around to give me a +1? Thanks

**Comment by arthurvr** [comment]
@agcolom Seems like that fixed it, thanks :)

**Comment by arschmitz** [comment]
LGTM :+1:

**Comment by agcolom** [comment]
@arthurvr @arschmitz Many thanks :-)

**Pull request closed without merging**